ai_serving/fasttext_clf/test.py

```diff
@@ -0,0 +1,15 @@
+from train import load_model
+from config import test_file_path
+from predict import predict
+
+if __name__ == '__main__':
+    model = load_model()
+
+    print('')
+    print('TESTING')
+    with open(test_file_path, 'r', encoding='utf-8') as input_file:
```

Comment:
Same with testing. I think all operations on the model, like train, test, predict should be member functions of the same class.

This way when we expose training op to jaseci it will be super easy.